packages/core/src/page/index.ts

```diff
@@ -1,9 +1,69 @@
-import { sendMessage } from '@polkadot/extension-base/page';
 
+import { PolyMessageTypes, PolyMessageTypesWithNoSubscriptions, PolyMessageTypesWithNullRequest, PolyMessageTypesWithSubscriptions, PolyRequestTypes, PolyResponseTypes, PolySubscriptionMessageTypes, PolyTransportRequestMessage, PolyTransportResponseMessage } from '../background/types';
```

Comment:
Instead of adding multiple lint disable lines, perhaps it would be less noisy to just disable specified lints for this file by adding them in the beginning:

```
1 |/* eslint-disable @typescript-eslint/no-explicit-any */
2 |/* eslint-disable-next-line @typescript-eslint/ban-types */